.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,3 +2,19 @@
 **Asana Ticket:** [TICKET_NAME](TICKET_LINK)
 
 [Please include a brief description of what was changed]
+
+---
+
+Before getting review, please check the following:
+
+* [ ] Does frontend functionality render and work correctly in IE?
+* [ ] Has every endpoint that will be polled from the frontend been load-tested?
```

Comment:
More generally, I would say "Have we load-tested any new pages or internal API endpoints that will receive significant traffic?" We don't introduce new pages or endpoints that often, and I think it'd be nice to consider this question whenever we do. The site does serve a large number of visitors with a small number of instances, so it needs to scale well in general.